lldb/include/lldb/Protocol/MCP/Protocol.h

```diff
@@ -198,12 +207,165 @@ struct ToolDefinition {
   // JSON Schema for the tool's parameters.
   std::optional<llvm::json::Value> inputSchema;
 };
-
 llvm::json::Value toJSON(const ToolDefinition &);
 bool fromJSON(const llvm::json::Value &, ToolDefinition &, llvm::json::Path);
 
 using ToolArguments = std::variant<std::monostate, llvm::json::Value>;
 
+/// Describes the name and version of an MCP implementation, with an optional
+/// title for UI representation.
+///
+/// see
```

Comment:
```suggestion
/// See
```
Or just drop all the "See"s because (1) the URL is pretty self explanatory and (2) there's no period after the URL.